src/d3a/d3a_core/simulation.py

```diff
@@ -305,12 +305,16 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
             log.debug(f"Used {mbs_used} MBs.")
 
             for tick_no in range(tick_resume, config.ticks_per_slot):
+                tick_start = time.monotonic()
```

Comment:
Why did you use monotonic clock here ? DST is not enabled in our services IIRC, therefore the monotonic clock will not be of use. Better to revert to the standard `time` method that we use elsewhere.

Reply:
Umm, tick_start was already using monotonic_clock so I continued the legacy and reused it to hold the simulation for certain time before stopping it in case the used the user didn't resume it.

Reply:
I see now that this was there right from the start. I would actually suggest to change this to `time.time()`. I think that there will be no problem with it, and then timing will be consistent across d3a.

Reply:
@faizan2590 I dont see Spyros' suggestion adapted. Was there an offline discussion about this?

Reply:
I need to incorporate it indeed but was waiting for your review to do it in one go